src/utils/mutate.js

```diff
@@ -58,11 +58,15 @@ const primaryValue = (arr) =>
 
 const arrayUnion = (firebase, key, val) => {
   if (key !== '::arrayUnion') return null;
-  return firebase.firestore.FieldValue.arrayUnion(val);
+  const arr = Array.isArray(val) ? val : [val];
+  return firebase.firestore.FieldValue.arrayUnion(...arr);
 };
 
-const arrayRemove = (firebase, key, val) =>
-  key === '::arrayRemove' && firebase.firestore.FieldValue.arrayRemove(val);
+const arrayRemove = (firebase, key, val) => {
+  if (key !== '::arrayRemove') return null;
+  const arr = Array.isArray(val) ? val : [val];
+  return firebase.firestore.FieldValue.arrayRemove(...arr);
+};
```

Comment:
What do you think about this?
```suggestion
const arrayRemove = (firebase, key, ...val) => {
  if (key !== '::arrayRemove') return null;
  return firebase.firestore.FieldValue.arrayRemove(...val);
};
```
It should handle both cases with one and more elements. And also it will be closer to the native one.

Reply:
Sure that works.

Reply:
Changes are up.